pkg/kubelet/sharedcpus/sharedcpus.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sharedcpus
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+)
+
+const (
+	sharedCpusFileName     = "/etc/kubernetes/openshift-shared-cpus"
+	sharedCpusResourceName = "workload.openshift.io/enabled-shared-cpus"
```

Comment:
We can call it `workload.openshift.io/shared-cpus`

Reply:
The value should be used as a boolean, hence the `enabled` at the prefix